Command and Control/DofoilNameCoinServerTraffic.txt

```diff
@@ -0,0 +1,11 @@
+// This is a query to retrieve last 60 days network connections to known Dofoil NameConin servers

```

Comment:
Could you please change the folder to something like “Campaigns”? Following up by email